definitions/mobile.d.ts

```diff
@@ -6,49 +6,92 @@ declare module Mobile {
 		skipRefresh?: boolean;
 	}
 
+	interface IDeviceInfo {
+		identifier: string;
+		displayName: string;
+		model: string;
+		version: string;
+		vendor: string;
+		platform: string;
+	}
+	
 	interface IDevice {
-		getIdentifier(): string;
-		getInstalledApplications(): IFuture<string[]>;
-		getDisplayName(): string;
-		getModel(): string;
-		getVersion(): string;
-		getVendor(): string;
-		getPlatform(): string;
+		deviceInfo: Mobile.IDeviceInfo;
+		applicationManager: Mobile.IDeviceApplicationManager;
+		fileSystem: Mobile.IDeviceFileSystem;
 		deploy(packageFile: string, packageName: string): IFuture<void>;
-		sync(localToDevicePaths: ILocalToDevicePathData[], appIdentifier: IAppIdentifier, liveSyncUrl: string): IFuture<void>;
-		sync(localToDevicePaths: ILocalToDevicePathData[], appIdentifier: IAppIdentifier, liveSyncUrl: string, options: ISyncOptions): IFuture<void>;
 		openDeviceLogStream(): void;
-		runApplication(applicationId: string): IFuture<void>;
-		uninstallApplication(applicationId: string): IFuture<void>;
-		listFiles(devicePath: string): IFuture<void>;
-		getFile(deviceFilePath: string): IFuture<void>;
-		putFile(localFilePath: string, deviceFilePath: string): IFuture<void>;
+	}
+	
+	interface IAndroidDevice extends IDevice {
+		adb: Mobile.IAndroidDebugBridge;		
+		sendBroadcastToDevice(action: string, extras: IStringDictionary): IFuture<number>;
 	}
 
-	interface IAppIdentifier {
+	interface IIOSDevice extends IDevice {
+		startService(serviceName: string): number;
+		mountImage(): IFuture<void>;
```

Comment:
`mountImage` internally finds the path to developer image - https://github.com/telerik/mobile-cli-lib/pull/376/files#diff-3efdd98dbb45e19bb229436517332d9dL310 or respects the `ddi` option. I think this is a better approach, because we actually don't know where the image is located. All the logic that finds the best image match is encapsulated in this function https://github.com/telerik/mobile-cli-lib/blob/fatme/ns-livesync/mobile/ios/ios-device.ts#L175